tools/release/release_utils.sh

```diff
@@ -66,6 +66,24 @@ function publishImages {
         echo "push image: $image"
         docker tag $image:$baseTag $image:$curTag
         retry 5 docker push $image:$curTag
+
```

Comment:
Is this line break based on some kind of standard? If it is not, please remove it.

Reply:
I'll remove that